in-market-test/v2/README.md

```diff
@@ -0,0 +1,110 @@
+# In-Market Test (2024) - Helper Party Runbook
+
+The runbook is set up to use [ansible](https://docs.ansible.com/ansible/latest/) to provision the remote machine. You can, of course, decide to run the commands in the ansible scripts manually if you prefer.
+
+## Local Install
+
+First, install [pipx](https://github.com/pypa/pipx?tab=readme-ov-file#install-pipx). On a Mac
+
+```
+brew install pipx
+```
+
+Then install ansible
+
+```
+pipx install --include-deps ansible
+```
+
+
+## Provisioning
+
+
+### Launching
+You'll need to provision a host on your cloud provider with roughly the following specs:
+1. 32 vCPU
+2. 64 GB memory
+3. 12.5 Gbps network bandwidth
+
+On AWS, we use c6id.8xlarge with the Amazon Linux 2023 AMI.
+
+The host needs SSH access (from you, if you want to limit the IP range), and HTTPS access from the internet (e.g., port 443 needs to be open.) No other ports need to be opened.
+
+
+### SSH access
+
+You'll need to configure the ansible inventory and your ssh config. Save the SSH key provided by the cloud provider to your ssh directory, e.g. `~/.ssh/helper_connect.pem`, then update `~/.ssh/config` to include:
+
+```
+Host ipa-helper
+  HostName <ipa-helper-ip-or-public-dns-name>
+  User <associated-user>
+  IdentityFile ~/.ssh/helper_connect.pem
+```
+
+Now, copy the inventory template
+
+```
+cp in-market-test/v2/ansible/inventory-template.ini in-market-test/v2/ansible/inventory.ini
```

Comment:
it is better to keep the templates and actual files used to deploy in different folders I think. Makes it easier to clean up and avoid committing local state. `ansible/templates/...` and `ansible/...` may be one way to do it